bcgov/HOUS-permit-portal
# Add notification service, controller, initialization, stores, UI, policy, etc

## Description

Add notifications system, fully featured with websocket update and pagination.

## What type of PR is this? (check all applicable)

- [ x] 🍕 Feature

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-205

## Steps to QA

Log in as review manager
See the notification area!

At the moment, events can only be generated through the console:

```
NotificationService.publish_new_template_version_publish_event(TemplateVersion.first)
```

Make sure the websocket queue and processes are running:
```
bundle exec sidekiq -t 25 -c 25 -q websocket
bundle exec anycable
anycable-go --port=8080
```
